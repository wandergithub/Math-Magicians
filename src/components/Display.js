import React from 'react';

class Display extends React.Component {
  constructor(props) {
    super(props);
    this.state = {
      output: 0,
    };
  }

  render() {
    const { output } = this.state;
    return (
      <div className="outputContainer">
        <p>{ output }</p>
      </div>
    );
  }
}

export default Display;
