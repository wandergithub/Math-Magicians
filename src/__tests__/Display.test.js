import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Display from '../components/Display';

describe('Display component tests', () => {
  it('Renders correctly', () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <Display />
        </MemoryRouter>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
