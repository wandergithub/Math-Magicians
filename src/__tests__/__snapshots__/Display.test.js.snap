// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display component tests Renders correctly 1`] = `
<div
  className="outputContainer"
>
  <p>
    0
  </p>
</div>
`;
